### ISSUE TITLE
D2BotMule issue when using all cd-keys to mf

### ISSUE THREAD

**Issue by GabrielDias1992** [kolton/d2bot-with-kolbot#911]
Using 3 cd-keys sets (1.1, 1.2 and 1.3) i have the following profiles:
- 1.2-Mule
- 1.3-Mule
- 1.1-Leader
- 1.2-Leader
- 1.3-Leader

And this is my AutoMule.js configuration:
```
Mules: {
	"Mule1":  {
		muleProfile: "1.3-Mule",  // The name of mule profile in d2bot#. It will be started and stopped when needed.
		accountPrefix: "accountPrefix",  // Account prefix. Numbers added automatically when making accounts.
		accountPassword: "accountPassword",  // Account password.
		charPrefix: "charPrefix",  // Character prefix. Suffix added automatically when making characters.
		realm: "useast", // Available options: "useast", "uswest", "europe", "asia"
		expansion: true,
		ladder: true,
		hardcore: false,

		// Game name and password of the mule game. Never use the same game name as for mule logger.
		muleGameName: ["muleGameName", "muleGameName"], // ["gamename", "password"]

		// List of profiles that will mule items. Example: enabledProfiles: ["profile 1", "profile 2"],
		enabledProfiles: ["1.1-Leader", "1.2-Leader"],

		// Stop a profile prior to muling. Useful when running 8 bots without proxies.
		stopProfile: "1.3-Leader",
		stopProfileKeyRelease: true, // true = stopProfile key will get released on stop. useful when using 100% of your keys for botting.

		// Trigger muling at the end of a game if used space in stash and inventory is equal to or more than given percent.
		usedStashTrigger: 80,
		usedInventoryTrigger: 80,

		// Mule items that have been stashed at some point but are no longer in pickit.
		muleOrphans: true
	},
	"Mule2":  {
		muleProfile: "1.2-Mule",  // The name of mule profile in d2bot#. It will be started and stopped when needed.
		accountPrefix: "accountPrefix",  // Account prefix. Numbers added automatically when making accounts.
		accountPassword: "accountPassword",  // Account password.
		charPrefix: "charPrefix",  // Character prefix. Suffix added automatically when making characters.
		realm: "useast", // Available options: "useast", "uswest", "europe", "asia"
		expansion: true,
		ladder: true,
		hardcore: false,

		// Game name and password of the mule game. Never use the same game name as for mule logger.
		muleGameName: ["muleGameName", "muleGameName"], // ["gamename", "password"]

		// List of profiles that will mule items. Example: enabledProfiles: ["profile 1", "profile 2"],
		enabledProfiles: ["1.3-Leader"],

		// Stop a profile prior to muling. Useful when running 8 bots without proxies.
		stopProfile: "1.2-Leader",
		stopProfileKeyRelease: true, // true = stopProfile key will get released on stop. useful when using 100% of your keys for botting.

		// Trigger muling at the end of a game if used space in stash and inventory is equal to or more than given percent.
		usedStashTrigger: 80,
		usedInventoryTrigger: 80,

		// Mule items that have been stashed at some point but are no longer in pickit.
		muleOrphans: true
	}
}
```

I want to:
- 1.3-Mule take the items from 1.1-Leader and 1.2-Leader.
- 1.2-Mule take the items from 1.3-Leader

But the code is not working, i am receiving the message: 21:57:46 (1.1-Leader) No response from mule profile.

Can someone help me ?

**Comment by 5noop** [comment]
What was happening when your received that error message, did "1.3-Leader" stop and "1.3-Mule" start ?

**Comment by GabrielDias1992** [comment]
The 1.3-Mule starts the game, then the 1.1-Leader stays in the lobby until the message:  (1.1-Leader) No response from mule profile.

After that the 1.3-Mule profile is closed and the 1.1-Leader goes back to the mfing run.

**Comment by GabrielDias1992** [comment]
I was using sojreporter to change the Diablo 2 window title, adding the game ip address.

This was the problem, you cannot edit the Diablo 2 window title.
